addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -148,7 +148,7 @@
         </field>
     </record>
 
-    <record id="hr_attendance_action_employee" model="ir.actions.act_window">
+    <record id="hr_attendance_action_employee" model="ir.actions.act_window"> <!-- Not used -->
```

Comment:
Why mark this record unused and make a copy? You could just modify the original action and reduce the diff. It won't affect databases that do not update the `hr_attendance` module, and when they do, everything will be updated.

Reply:
Because a customization using this view would be impacted by these changes. I've tried to override the view's context in the button, but it doesn't work that way.

Reply:
I preferred not to have view used only for attendance office be used for seeing your own attendance (my profile -> attendance) and that other people that used the view expecting it to filter on active_id don't get it non-filtered all of a sudden.

If it's okay we could change `hr_attendance_action_employee` as done initially.

I'm not sure the change in view_employee_form_inherit_hr_attendance:

```diff
-                 <button name="%(hr_attendance_action)d"
+                <button name="%(hr_attendance_action_no_create)d"
```

is necessary since we only see that button when we are an attendance officer.

Reply:
> Because a customization using this view would be impacted by these changes. I've tried to override the view's context in the button, but it doesn't work that way.

It's great that you're trying to avoid break customisations, but in this case I find it overkill:

1) The chances of having a customization impacted by this change are very small, because the change is tiny (just the `context` AFAICS). What could break? It's very hard for a customisation to hard-depend on that. The behavior will be sightly different, but that's the whole point of the fix.
2) The change won't impact any database unless they force-update the module, or go through an upgrade. At that point, so many things can break in terms of customisations, and this one is a very minor concern.
3) By introducing a new XML ID, you *also* add some risk: for a future bugfix in another module, say e.g. `hr_holidays_attendance`, you may be tempted to use this new XML ID. But now if someone installs module `hr_holidays_attendance` while they have never updated `hr_attendance`, it will crash (unknown XML ID).
4) Finally, by duplicating the action you're introducing some tech debt in the codebase, as someone will have to cleanup the dead code later (or it will be forgotten).

Long story short, as with all things, common sense prevails, even with the stable policy. Think in terms of the number of customer databases that will be possibly affected by the change (here: I can hardly imagine a single one), and the severity of the problem they will experience if that ever happens (here: very minor behavior change), and choose the option with the best risk/benefit ratio :)